au4/src/trackedit/internal/au3/au3trackeditproject.h

```diff
@@ -32,6 +35,11 @@ class Au3TrackeditProject : public ITrackeditProject
 
     secs_t totalTime() const override;
 
+    bool undoAvailable() override;
```

Comment:
It seems that it is better to move this to a separate class and get it from Audacity4Project
it's a different object than simple operations on tracks/clips